.github/workflows/conda.yml

```diff
@@ -72,8 +72,20 @@ jobs:
         env:
           ANACONDA_API_TOKEN: ${{ secrets.DASK_CONDA_TOKEN }}
         run: |
+          # convert distributed to other architectures
+          cd build && conda convert linux-64/*.tar.bz2 -p osx-64 \
+                                                       -p osx-arm64 \
+                                                       -p linux-ppc64le \
+                                                       -p linux-aarch64 \
+                                                       -p win-64
+
```

Comment:
Ideally we should keep distributed nightlies `noarch`, so that we don't need to convert the package to different architectures before uploading:

```suggestion
```